packages/widget/vite.preview.config.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+import { resolve } from 'path';
+
+// https://vitejs.dev/config/
+export default defineConfig({
```

Comment:
Overall, it looks good! Can you please provide the information about improvement? Why is it better to use a preview build?